### ISSUE TITLE
prometheus community contribution

### ISSUE THREAD

**Issue by kfox1111** [Sheridan/smartctl_exporter#27]
Would you be interested in contributing this to to the prometheus community project? There seems to be some interest:
https://github.com/prometheus-community/community/issues/3#issuecomment-899641944

Thanks,
Kevin

**Comment by frittentheke** [comment]
> Would you be interested in contributing this to to the prometheus community project? There seems to be some interest:
> [prometheus-community/community#3 (comment)](https://github.com/prometheus-community/community/issues/3#issuecomment-899641944)
Thanks@kfox1111 for bringing this idea up!

@Sheridan IMHO it would be great to join the SMART exporting efforts (see my comment at https://github.com/prometheus-community/community/issues/3#issuecomment-898459512) and to focus the development less on various different implementations.

Having a smartctl exporter be part of prometheus-community would certainly be a great kick-off!

**Comment by Sheridan** [comment]
Yes, it will be good.  I'll be back from vacation and do what I need to do.  By the way, what should be done?  :)

**Comment by frittentheke** [comment]
@Sheridan would you still be willing to move your smartctl_exporter to be a prometheus-community project (https://github.com/prometheus-community/community/issues/3)? @superq apparently would take this exporter on board - see https://github.com/prometheus-community/community/issues/3#issuecomment-906525853

Would be awesome to have somewhat of a reference implementation to this quite commonly collected metrics.
I suppose you don't more some more helping hands anyways?

**Comment by kfox1111** [comment]
@frittentheke I think the ball is in the prometheus-community court, in the other thread.
